app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>EDIT A TASK</h2>
+
+<%= form_with model: @task, class: 'create-task' do |f| %>
+  <p>Please provide the following information to edit the task to our database:</p>
+
+  <div>
+    <%= f.label :name %>:
+    <%= f.text_field :name %>
+  </div>
+
+  <div>
+    <%= f.label :description %>:
+    <%= f.text_field :description %>
+  </div>
+
+  <div>
+    <%= f.label :completed_at %>:
+    <%= f.text_field :completed_at %>
+  </div>
```

Comment:
Consider leaving this off from your form as it would make sense that new task has set `nil` for `completed_at`, and you have a custom method to change this (`mark_complete` and `mark_uncomplete`).